### ISSUE TITLE
in2csv for ndjson breaks when first line contains "yesterday"

### ISSUE THREAD

**Issue by pokey** [wireservice/csvkit#1094]
```sh
➜ in2csv --version
in2csv 1.0.5

➜ printf '{"foo":"yesterday"}' | in2csv -f ndjson
OverflowError: date value out of range

➜ printf '{"foo":"hello"}\n{"foo":"yesterday"}' | in2csv -f ndjson
foo
hello
yesterday
```

**Comment by pokey** [comment]
Fwiw it looks like a workaround is to use the `--no-inference` flag

**Comment by jpmckinney** [comment]
Hmm, it's because the sourceTime given to parsedatetime by agate is the minimum date, and "yesterday" puts that beyond the minimum. It's from a very old commit: https://github.com/wireservice/agate/commit/eded6f5440fe63e2f3c6a9fbd2fb78b0430839c0 I don't know why the sourceTime isn't today (it'd make sense for "yesterday", but maybe not for other times).

**Comment by jpmckinney** [comment]
I've released agate 1.6.2 which should fix this: https://github.com/wireservice/agate/pull/720
